### PR TITLE
rename several shadowed variables

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/metadata/exiv2_handler.cc [hunk 1]
```diff
@@ -137,29 +137,29 @@ void Exiv2Handler::fillMetadata(const std::shared_ptr<CdsObject>& item)
...
                 log_debug("auxtag: {} ", auxtag.c_str());
                 if (auxtag.substr(0, 4) == "Exif") {
-                    const auto md = exifData.findKey(Exiv2::ExifKey(auxtag));
+                    md = exifData.findKey(Exiv2::ExifKey(auxtag));
                     if (md != exifData.end())
-                        value = md->toString();
+                        val = md->toString();
                 } else if (auxtag.substr(0, 3) == "Xmp") {
-                    const auto md = xmpData.findKey(Exiv2::XmpKey(auxtag));
+                    auto md = xmpData.findKey(Exiv2::XmpKey(auxtag));
```
Comment (KarlStraussberger) [inline src/metadata/exiv2_handler.cc:152]: You don't need `auto` here either.

File: src/metadata/exiv2_handler.cc [hunk 2]
```diff
@@ -137,29 +137,29 @@ void Exiv2Handler::fillMetadata(const std::shared_ptr<CdsObject>& item)
         // if there are any auxilary tags that the user wants - add them
         const auto aux = config->getArrayOption(CFG_IMPORT_LIBOPTS_EXIV2_AUXDATA_TAGS_LIST);
         if (!aux.empty()) {
-            std::string value;
+            std::string val;
```
Comment (KarlStraussberger) [inline src/metadata/exiv2_handler.cc:140]: I'd prefer `auxval`

File: src/metadata/exiv2_handler.cc [hunk 3]
```diff
@@ -137,29 +137,29 @@ void Exiv2Handler::fillMetadata(const std::shared_ptr<CdsObject>& item)
...
             std::string auxtag;
 
             for (auto&& j : aux) {
```
Comment (KarlStraussberger) [inline src/metadata/exiv2_handler.cc:143]: `j` can become `auxtag` to drop the outer loop declaration.

File: src/metadata/exiv2_handler.cc (hunk 1 above)
Comment (neheb) [reply src/metadata/exiv2_handler.cc:152]: Yeah you do. xmpData is a difference type than ExifData.

Comment (KarlStraussberger) [reply src/metadata/exiv2_handler.cc:152]: Then choose another name like `xmpMd`.

**Pull request merged**